### PR TITLE
@aadattoli additional changes to diagnostics for merge

### PR DESCRIPTION
**Pull request by rando2**
@rando2 needs to fix this

### CODE REVIEW

File: content/10.diagnostics.md [hunk 1]
```diff
@@ -7,47 +7,34 @@ For instance, the capability of a large scale diagnostic test and the following
...
 Particularly, this test detects several regions of the viral genome: orf1b of the RNA dependent RNA polymerase (RdRP), Envelope protein gene (E), Nucleocapsid protein gene (N). 
 The lab, which collaborated with several other labs in Europe and in China, confirmed the specificity of this test, with the respect of other corona viruses, against specimens from 297 patients with a broad range of respiratory agents. 
 Specifically this tests utilises two probes against RdRP of which one is specific for COVID-19 [doi.org/10.2807/1560-7917.ES.2020.25.3.2000045]. Importantly, this assay did not give any false positive outcome.
 
-QRT-tPCR
+2) qRT-tPCR
 
-Around the same time the Hong Kong university developed a Quantitative Real Time Reverse Transcription PCR (QRT-tPCR) against two region of the virus, orf1b and N region [doi.org/10.1093/clinchem/hvaa029]. 
+Around the same time the Hong Kong university developed a Quantitative Real Time Reverse Transcription PCR (qRT-tPCR) against two region of the virus, orf1b and N region [doi.org/10.1093/clinchem/hvaa029].
```
Comment (cbrueffer) [inline content/10.diagnostics.md:23]:
```suggestion
Around the same time the University of Hong Kong developed quantitative real-time reverse transcription PCR (qRT-PCR) assays against two region of the virus, orf1b and N region [@doi:10.1093/clinchem/hvaa029]. 
```

File: content/10.diagnostics.md [hunk 2]
```diff
@@ -7,47 +7,34 @@ For instance, the capability of a large scale diagnostic test and the following
...
-The RNA guidance helps CAS12 to direct the cut of a sequence corresponding to a piece of the coronavirus genome. 
-With this test, If CAS12 and its guide find a match in the RNA, then the cut occurs and the patient is positive to the novel Coronavirus. Specifically, when CAS12 is active it also cuts a designed strand containing molecules that change color once they are free [doi.org/10.1101/2020.03.06.20032334. 
-This test can be as fast as 30 minutes and the result can be as simple as a color change on a strip [doi:10.1126/science.abb8400].
+3) CRISPR test
 
-Limitation of CRISPR test
-
-Fake-positives due to the specificity of the technique, as the guide RNA can recognise other interspersed sequences on the patient’s genome.
+Recently, two American companies, Mammoth Biosciences and Sherlock Biosciences, proposed to use CRISPR technology [10.1016/j.cell.2014.05.010]to implement the speed of Coronavirus diagnostic test [doi.org/10.1101/2020.03.06.20032334; doi: 10.1038/s41596-019-0210-2].
```
Comment (cbrueffer) [inline content/10.diagnostics.md:32]:
```suggestion
Recently, two American companies, Mammoth Biosciences and Sherlock Biosciences, proposed to use CRISPR technology [@doi:10.1016/j.cell.2014.05.010] to implement the speed of Coronavirus diagnostic test [@doi:10.1101/2020.03.06.20032334; @doi: 10.1038/s41596-019-0210-2]. 
```

File: content/10.diagnostics.md [hunk 3]
```diff
@@ -7,47 +7,34 @@ For instance, the capability of a large scale diagnostic test and the following
...
+3) CRISPR test
 
-Limitation of CRISPR test
-
-Fake-positives due to the specificity of the technique, as the guide RNA can recognise other interspersed sequences on the patient’s genome.
+Recently, two American companies, Mammoth Biosciences and Sherlock Biosciences, proposed to use CRISPR technology [10.1016/j.cell.2014.05.010]to implement the speed of Coronavirus diagnostic test [doi.org/10.1101/2020.03.06.20032334; doi: 10.1038/s41596-019-0210-2]. 
+Their methodology includes purification of RNA extracted from patient specimen; loop-mediated iso-thermal amplification (fast amplification) of the RNA; addition of the CRISPR reagents: CAS12, which cut the DNA or RNA, and the RNA guidance. 
+The RNA guidance helps CAS12 to direct the cut of a sequence corresponding to a piece of the coronavirus genome (specifically, N, E and RdRP genes in the test designed by Mammoth). 
+With this test, If CAS12 and its guide find a match in the RNA, then the cut occurs and the patient is positive to the novel Coronavirus. Specifically, when CAS12 is active it also cuts a designed strand containing molecules that change color once they are free [doi.org/10.1101/2020.03.06.20032334].
```
Comment (cbrueffer) [inline content/10.diagnostics.md:35]:
```suggestion
With this test, if CAS12 and its guide find a match in the RNA, then the cut occurs and the patient is positive to the novel Coronavirus. Specifically, when CAS12 is active it also cuts a designed strand containing molecules that change color once they are free [@doi:10.1101/2020.03.06.20032334]. 
```

File: content/10.diagnostics.md [hunk 4]
```diff
@@ -7,47 +7,34 @@ For instance, the capability of a large scale diagnostic test and the following
...
+Their methodology includes purification of RNA extracted from patient specimen; loop-mediated iso-thermal amplification (fast amplification) of the RNA; addition of the CRISPR reagents: CAS12, which cut the DNA or RNA, and the RNA guidance. 
+The RNA guidance helps CAS12 to direct the cut of a sequence corresponding to a piece of the coronavirus genome (specifically, N, E and RdRP genes in the test designed by Mammoth). 
+With this test, If CAS12 and its guide find a match in the RNA, then the cut occurs and the patient is positive to the novel Coronavirus. Specifically, when CAS12 is active it also cuts a designed strand containing molecules that change color once they are free [doi.org/10.1101/2020.03.06.20032334]. 
+Mammoth CRISPR test is designed to give a practical solution in area that are at greater risk of infection (i.e. airports, local community hospitals). This test shows a sensitivity range of 70-300 copies of the target RNA/ul.
```
Comment (cbrueffer) [inline content/10.diagnostics.md:36]:
```suggestion
Mammoth CRISPR test is designed to give a practical solution in areas that are at greater risk of infection (i.e. airports, local community hospitals). This test shows a sensitivity range of 70-300 copies of the target RNA/ul. 
```

File: content/10.diagnostics.md [hunk 5]
```diff
@@ -57,14 +44,14 @@ In fact, they can help scientists figuring out why the disease has different cou
...
 When the patient's specimen contains antibody against COVID-19 antigens they will bind to them and capture either by the IgM and or IgG line, with the result of a change of color [https://www.fda.gov/media/136625/download]. 
-With this particular assay results can be read within 15-20 minutes [https://www.fda.gov/media/136625/download]. 
-Importantly, false-positives can occur due to the cross-reactivity with other antibodies according to the clinical condition of the patient [https://www.fda.gov/media/136625/download]. 
-Therefore, this test should be used in combination with those RNA detection tests [https://www.fda.gov/media/136625/download]. 
+With this particular assay results can be read within 15-20 minutes [https://www.fda.gov/media/136625/download].
+Other research groups, such as the Krammer lab of the Icahn School of Medicine at Mount Sinai proposed an ELISA test that detects IgG and IgM that react against the receptor binding domain (RBD) of the spike proteins (S) of the virus [doi.org/10.1101/2020.03.17.20037713].
```
Comment (cbrueffer) [inline content/10.diagnostics.md:53]:
```suggestion
Other research groups, such as the Krammer lab of the Icahn School of Medicine at Mount Sinai proposed an ELISA test that detects IgG and IgM that react against the receptor binding domain (RBD) of the spike proteins (S) of the virus [@doi:10.1101/2020.03.17.20037713]. 
```

File: content/10.diagnostics.md [hunk 6]
```diff
@@ -79,9 +66,40 @@ Therefore, this test should be used in combination with those RNA detection test
...
 ### Limitations to Implementation of Large-Scale Testing
 
-[Right now, reagent supply is an issue. Are there others concerns that are likely to emerge?]
+Limitation of molecular tests
+
+-PCR tests
+
+The PCR test can be highly sensitive. However, in high-throughput settings it can show several problems:
+
+1)	 False-negative responses, which can be a significant problem in a large scale test. In this case a correct execution is crucial [doi: 10.1038/d41587-020-00002-2].
```
Comment (cbrueffer) [inline content/10.diagnostics.md:75]:
```suggestion
1)	 False-negative responses, which can be a significant problem in a large scale test. In this case a correct execution is crucial [@doi:10.1038/d41587-020-00002-2]. 
```

File: content/10.diagnostics.md [hunk 7]
```diff
@@ -79,9 +66,40 @@ Therefore, this test should be used in combination with those RNA detection test
...
-[Right now, reagent supply is an issue. Are there others concerns that are likely to emerge?]
+Limitation of molecular tests
+
+-PCR tests
+
+The PCR test can be highly sensitive. However, in high-throughput settings it can show several problems:
+
+1)	 False-negative responses, which can be a significant problem in a large scale test. In this case a correct execution is crucial [doi: 10.1038/d41587-020-00002-2]. 
+
+2)	Uncertainty of the SARS-Cov-2 viral shedding kinetics, which may affect the result of test if not taken at the right time [doi: 10.1038/d41587-020-00002-2].
```
Comment (cbrueffer) [inline content/10.diagnostics.md:77]:
```suggestion
2)	Uncertainty of the SARS-CoV-2 viral shedding kinetics, which may affect the result of the test if not taken at the right time [@doi:10.1038/d41587-020-00002-2].
```

File: content/10.diagnostics.md [hunk 8]
```diff
@@ -79,9 +66,40 @@ Therefore, this test should be used in combination with those RNA detection test
...
+
+3)	Type of Specimen, as it is not clear which clinical sample is best to detect the virus [doi: 10.1038/d41587-020-00002-2].
```
Comment (cbrueffer) [inline content/10.diagnostics.md:79]:
```suggestion
3)	Type of Specimen, as it is not clear which clinical sample is best to detect the virus [@doi:10.1038/d41587-020-00002-2].
```

File: content/10.diagnostics.md [hunk 9]
```diff
@@ -79,9 +66,40 @@ Therefore, this test should be used in combination with those RNA detection test
...
+
+4)	Expensive machinaries, which might be present in major hospital and or diagnostic centers, but not in smaller facilities [doi:10.1126/science.abb8400].
```
Comment (cbrueffer) [inline content/10.diagnostics.md:81]:
```suggestion
4)	Expensive machinary, which might be present in major hospital and diagnostic centers, but not in smaller facilities [@doi:10.1126/science.abb8400]. 
```

File: content/10.diagnostics.md [hunk 10]
```diff
@@ -79,9 +66,40 @@ Therefore, this test should be used in combination with those RNA detection test
...
+
+1)	 False-negative responses, which can be a significant problem in a large scale test. In this case a correct execution is crucial [doi: 10.1038/d41587-020-00002-2]. 
+
+2)	Uncertainty of the SARS-Cov-2 viral shedding kinetics, which may affect the result of test if not taken at the right time [doi: 10.1038/d41587-020-00002-2].
+
+3)	Type of Specimen, as it is not clear which clinical sample is best to detect the virus [doi: 10.1038/d41587-020-00002-2].
+
+4)	Expensive machinaries, which might be present in major hospital and or diagnostic centers, but not in smaller facilities [doi:10.1126/science.abb8400]. 
+
+5)	Timing of the test, which might take up to 4 days, due to shipping of collected samples, to give results [doi:10.1126/science.abb8400].
```
Comment (cbrueffer) [inline content/10.diagnostics.md:83]:
```suggestion
5)	Timing of the test, which might take up to 4 days, due to shipping of collected samples, to give results [@doi:10.1126/science.abb8400].
```

File: content/10.diagnostics.md [hunk 11]
```diff
@@ -7,47 +7,34 @@ For instance, the capability of a large scale diagnostic test and the following
 Following the first release of the genetic sequence of the virus by Chinese officials on January 10 2020, the first test was released about 13 days later [@doi:10.2807/1560-7917.ES.2020.25.3.2000045]. 
 Testing who contracted the infection is crucial to slow down the global pandemic. The available assays use the following approach to identify the active virus in patient samples.
 
-RT-PCR 
+Molecular tests
+
+Molecular tests are often carried out from the nasopharyngeal cavity samples, including throat wash and saliva [doi.org/10.1016/S1473-3099(20)30196-1].
```
Comment (cbrueffer) [inline content/10.diagnostics.md:12]:
```suggestion
Molecular tests are often carried out from the nasopharyngeal cavity samples, including throat wash and saliva [@doi:10.1016/S1473-3099(20)30196-1].
```

File: content/10.diagnostics.md (hunk 6 above)
Comment (cbrueffer) [inline content/10.diagnostics.md:75]:
```suggestion
1)	 False-negative responses, which can be a significant problem in a large scale test. In this case a correct execution is crucial [@doi:10.1038/d41587-020-00002-2]. 
```

File: content/10.diagnostics.md [hunk 12]
```diff
@@ -79,9 +66,40 @@ Therefore, this test should be used in combination with those RNA detection test
...
+Limitation of seriological tests
+
+Importantly, false-positives can occur due to the cross-reactivity with other antibodies according to the clinical condition of the patient [https://www.fda.gov/media/136625/download]. 
+Therefore, these tests should be used in combination with RNA detection tests [https://www.fda.gov/media/136625/download]. 
 
 ### Strategies and Considerations for Determining Whom to Test
 
-[If it's not possible to test everyone, what strategies exist for selecting who to test?]
-[Are these strategies likely to change over time? Presumably there are different stages of managing spread vs mitigating severity once it's already at high prevalence?]
+Currently, Coronavirus tests are limited for people that are in danger of serious illness [https://www.cdc.gov/coronavirus/2019-nCoV/hcp/clinical-criteria.html]. Specififcally, the individuals at risk include:
```
Comment (cbrueffer) [inline content/10.diagnostics.md:98]:
```suggestion
Currently, Coronavirus tests are limited to people that are in danger of serious illness [https://www.cdc.gov/coronavirus/2019-nCoV/hcp/clinical-criteria.html]. Specifically, the individuals at risk include:
```

File: content/10.diagnostics.md [hunk 13]
```diff
@@ -79,9 +66,40 @@ Therefore, this test should be used in combination with those RNA detection test
...
+
+-people with severe symptoms
+-people showing mild symptomps that have been in contact with a person who has developed the infection
```
Comment (cbrueffer) [inline content/10.diagnostics.md:101]:
```suggestion
-people showing mild symptoms that have been in contact with a person who has developed the infection
```

File: content/10.diagnostics.md [hunk 14]
```diff
@@ -79,9 +66,40 @@ Therefore, this test should be used in combination with those RNA detection test
...
+-people with underlying health conditions
+-pregnant women
+
+However, this methodology of testing administration does not detect a high proportion of infections. This includes individuals that are asymptomatic (potentially able to infect others) and individuals that are able to recover at home. Furthermore, individuals for whom the origin of the infection is uncertain are not tested. For instance, a recent study from the Imperial College estimates that in Italy the true number of infections is around 5.9 million against the 70000 detected as of March 28th [doi.org/10.25561/77731].
```
Comment (cbrueffer) [inline content/10.diagnostics.md:105]:
```suggestion
However, this methodology of testing administration does not detect a high proportion of infections. This includes individuals that are asymptomatic (potentially able to infect others) and individuals that are able to recover at home. Furthermore, individuals for whom the origin of the infection is uncertain are not tested. For instance, a recent study from the Imperial College estimates that in Italy the true number of infections is around 5.9 million against the 70000 detected as of March 28th [@doi:10.25561/77731].   
```

**Comment by rando2** [comment]
https://github.com/aadattoli/covid19-review/tree/patch-4 uses an out-of-date version of content/10.diagnostics.md
I attempted to merge the two versions on the command line but need to inspect more closely -- proceed with caution if you're interested in editing this content!

**Comment by cbrueffer** [comment]
OK, I'll submit a new PR with my changes.

**Comment by rando2** [comment]
Thanks @cbrueffer, this text got submitted in a weird way, hopefully no more issues will come from it. Thanks for your feedback and sorry for the mess!

**Pull request merged**